### PR TITLE
Remove empty space in kubelet_config.json

### PR DESCRIPTION
**Pull request by jrvaldes**
Fix the template field for clientCAFile in the kubelet_config.json, by
removing an empty space, causing kubelet to ignore the CA certificate file.

### CODE REVIEW

**Comment by selansen** [comment]
certificate rotation is not a bug fix. I am fine with backporting it.

**Comment by sebsoto** [comment]
@jrvaldes held until cherry pick discussion resolved

**Comment by jrvaldes** [comment]
> @jrvaldes held until cherry pick discussion resolved

No backports. Deleted the /cherry-pick commands as per https://github.com/kubernetes/test-infra/issues/17174#issuecomment-622214376

File: pkg/bootstrapper/bootstrapper_test.go [hunk 1]
```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-//cniTest holds the location of the directories and files required for running some of the CNI tests
+// cniTest holds the location of the directories and files required for running some of the CNI tests
```
Comment (aravindhp) [inline pkg/bootstrapper/bootstrapper_test.go:15]: Formatting  fixes shouldn't be done in a bug fix commit. This has been called out a few times.

File: pkg/bootstrapper/bootstrapper_test.go [hunk 2]
```diff
@@ -235,7 +235,7 @@ func TestCreateKubeletConf(t *testing.T) {
 //	"excludeMasterFromStandardLB": null,
 //	"disableOutboundSNAT": null,
 //	"maximumLoadBalancerRuleCount": 0
-//}
+// }
```
Comment (aravindhp) [inline pkg/bootstrapper/bootstrapper_test.go:238]: Same here.

**Review by aravindhp — CHANGES_REQUESTED** [review]

**Review by aravindhp — COMMENTED** [review]
LGTM

**Comment by jrvaldes** [comment]
/test e2e-wmcb

Error: creating EC2 VPC: VpcLimitExceeded: The maximum number of VPCs has been reached.

**Comment by jrvaldes** [comment]
/hold

to investigate CI issue

**Comment by jrvaldes** [comment]
/hold 

Still getting AWS quota  [issues](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_windows-machine-config-bootstrapper/344/pull-ci-openshift-windows-machine-config-bootstrapper-master-e2e-wmcb/1508522945788514304)

**Comment by jrvaldes** [comment]
/hold cancel

CI issues with the AWS quota limit is resolved.

**Pull request merged**